serverless/serverless-getting-started.adoc

```diff
@@ -0,0 +1,30 @@
+[id="serverless-getting-started"]
+= Getting started with {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-getting-started
+
+:FeatureName: OpenShift Serverless
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+{ServerlessProductName} makes it simpler to get code from development into production by reducing the need for infrastructure set up or back-end development by developers.
```

Comment:
Consider rewording this a bit to help with translation:
{ServerlessProductName} simplifies the process of delivering code from development into production 
(or something like that  :thinking: )